### DISCUSSION TITLE
Pass the 'fontproperties' parameter to Matplotlib in nx_pylab  `draw_networkx_edge_labels`

### DISCUSSION THREAD

**Discussion by asdf3601a** [networkx/networkx#6656] [General]
Doing so would be helpful in adjusting the font better when displaying in Chinese characters.

**Comment by rossbar** [comment]
Adding `matplotlib` plotting options as explicit keyword arguments to the nx drawing functions so they can be passed through to matplotlib is a common flavor of request. Generally speaking, this is not a great pattern as it would mean that NetworkX would have to duplicate a very large portion of the `matplotlib` API in order to achieve features.

The recommended solution is to use the Matplotlib directly to tweak/manage/modify the visualization properties. For example:

```python
>>> G = nx.Graph([(0, 1)])
>>> pos = {n: (n, n) for n in G}
>>> fig, ax = plt.subplots()
>>> nx.draw_networkx_edges(G, pos, ax=ax)
>>> nx.draw_networkx_edge_labels(G, pos, ax=ax, edge_labels={(0, 1): r"$\lambda$"})
>>> for t in ax.texts:
...     t.set_fontproperties(...)
```
